sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -988,13 +988,13 @@ def from_string(cls, permission):
         p_permanent_delete = 'y' in permission
         p_list = 'l' in permission
         p_tag = 't' in permission
-        p_find = 'f' in permission
+        p_filter_by_tags = 'f' in permission
         p_move = 'm' in permission
         p_execute = 'e' in permission
         p_set_immutability_policy = 'i' in permission
         parsed = cls(read=p_read, write=p_write, delete=p_delete, list=p_list,
                      delete_previous_version=p_delete_previous_version, tag=p_tag, add=p_add,
-                     create=p_create, permanent_delete=p_permanent_delete, find=p_find, move=p_move,
+                     create=p_create, permanent_delete=p_permanent_delete, find=p_filter_by_tags, move=p_move,
```

Comment:
Shouldn't this be `filter_by_tags=p_filter_by_tags`?

Reply:
Good catch! Will update pending response to other comment :)